tools/goctl/api/cmd.go

```diff
@@ -47,6 +48,12 @@ var (
 		RunE:  gogen.GoCommand,
 	}
 
+	goCliCmd = &cobra.Command{
+		Use:   "gocli",
```

Comment:
In my opinion, it's recommend to move this sub-command under the `go` sub-command